--- drivers/usb/gadget/function/storage_common.c.updated        2024-10-13 15:07:41.984468608 +0800
+++ drivers/usb/gadget/function/storage_common.c   2024-10-13 14:14:30.729647763 +0800
@@ -244,12 +244,6 @@
        min_sectors = 1;
        if (curlun->cdrom) {
                min_sectors = 300;      /* Smallest track is 300 frames */
-               if (num_sectors >= 256*60*75) {
-                       num_sectors = 256*60*75 - 1;
-                       LINFO(curlun, "file too big: %s\n", filename);
-                       LINFO(curlun, "using only first %d blocks\n",
-                                       (int) num_sectors);
-               }
        }
        if (num_sectors < min_sectors) {
                LINFO(curlun, "file too small: %s\n", filename);
